Eshop.Persistence/ProductConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Eshop.Domain;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Eshop.Persistence
+{
+    internal class ProductConfiguration : IEntityTypeConfiguration<Product>
+    {
+        public void Configure(EntityTypeBuilder<Product> builder)
+        {
+            builder.HasNoKey();
+            builder.Property(x => x.Id);
+            builder.Property(x => x.Title);
+            builder.Property(x => x.Description);
+            builder.Property(x => x.Price);
+            builder.Ignore(x => x.Category);
```

Comment:
lol, ignore? really
This was exactly the place, where I was interested how you will solve it ;).
Use chatgpt to ask how to solve it. As much as you describe the problem even detailed then chatgpt will gives you answer.

Reply:
fixed, calling products by category props as well, tested